gratipay/utils/icons.py

```diff
@@ -0,0 +1,10 @@
+
+
+def statusicons():
```

Comment:
Doesn't even need to be a function, can just be a top-level constant, in which case all-caps is customary.

```
STATUS_ICONS = {}
```